nixos/doc/manual/release-notes/rl-2103.xml

```diff
@@ -466,6 +466,11 @@ self: super:
      ALSA OSS emulation (<varname>sound.enableOSSEmulation</varname>) is now disabled by default.
     </para>
    </listitem>
+   <listitem>
+     <para>
+       <xref linkend="opt-services.xserver.videoDrivers" /> no longer uses the deprecated <literal>cirrus</literal> and <literal>vesa</literal> device dependent X drivers by default. It also enables both amdgpu and nouveau drivers by default now.
+     </para>
+   </listitem>
```

Comment:
I would make this slightly more complete:

>  `opt-services.xserver.videoDriver` no longer sets deprecated `cirrus` and `vesa` drivers, but now enables both `amdgpu` and `nouveau` drivers by default.

This way, for people that used to put `amdgpu` or `nouveau` in their configuration, they can remove it if they want.

Reply:
Just add `<literal>amdgpu</literal>` and `<literal>nouveau</literal>` and you're good to go.

Reply:
done